### ISSUE TITLE
Uncaught SyntaxError: Unexpected token  in JSON

### ISSUE THREAD

**Issue by dioni21** [vozlt/nginx-module-vts#149]
Hi,

We use VTS in many of our servers, but I just caught this one.

In a server with `server_name` with metachar `*`, each new server name appears as `NO $host` in the Server Zones table.  But it appears that the separator char between `NO` and `$host` is invalid in a JSON, at least for Chrome 70.0.3538.110.

The following error appears in console:

```
VM7075:1 Uncaught SyntaxError: Unexpected token  in JSON at position 365
    at JSON.parse (<anonymous>)
    at XMLHttpRequest.b.onload (status:1)
```
I saw issue #131, but even using the module with commit 1a01a87 will not fix this.

I could "fix" by changing definition of `NGX_HTTP_VHOST_TRAFFIC_STATUS_KEY_SEPARATOR` from `0x1f` to `0x20`, but I'm not sure if this will break anything else.

**Comment by dioni21** [comment]
BTW: If anybody try to compile a version with the above change, remember to delete `vts.db` if using it!  I wasted a full day trying to debug such a system.  :sob:

**Comment by dioni21** [comment]
The good side: I found code to ESCAPE such char when exporting to JSON (`ngx_http_vhost_traffic_status_escape_json_pool`), but somehow it was not working?

**Comment by markblackman** [comment]
We're seeing this as well, that odd 01xf character ,"Unit Separator" is not part of the JSON specification as far as I can tell.  Not sure what the trigger is though, we don't have a '*', just some regex chars.

**Comment by u5surf** [comment]
Hi, @dioni21 @markblackman 
We're sorry to be late comment,
I'm trying to fix the same issue below, but we cannot reproduce only one pattern which the server_name include the UNIT SEPARATOR like this. We would like to work on this issue, we need to know how to reproduce properly as much as possible.

https://github.com/vozlt/nginx-module-vts/issues/212#issuecomment-1240223917

Can you help to fix this issue?
Can you share your nginx.conf which become the behavior, and also it can reproduce even if it is the latest version?

**Comment by u5surf** [comment]
@dioni21 Hi, can you check your nginx build configure-arguments?
If there had `—without-http-cache`, it could be a bug obviously.

We could find the reproduction below
https://github.com/vozlt/nginx-module-vts/issues/212#issuecomment-1249138388

It’ll be fixed with #246
